tests/test_coordinator.py

```
@@ -0,0 +1,53 @@
+
+
+@pytest.mark.asyncio
+async def test_async_update_data_success(hass, myair_client):
+    coordinator = MyAirDataUpdateCoordinator(hass, myair_client)
+    data = await coordinator._async_update_data()
+    assert data["device"] == {"serial": "1234"}
+    assert data["sleep_records"] == [{"totalUsage": 60}]
+
+
+@pytest.mark.asyncio
+async def test_async_update_data_auth_error(hass, myair_client):
```

Ruff (0.12.2)

Missing docstring in public function

(D103)